TCL4 Data Management/utm-tcl4-dmp-con1.yaml

```diff
@@ -415,13 +364,13 @@ definitions:
     required: ['temperature','pressure','windSpeed','windDir']
     description: 4 numbers for temperature, air pressure (inHg), windspeed (ft/sec), wind direction
     properties:
-      temp:
+      temperature:
```

Comment:
Assuming this is Celsius given your bounds.  If truly a number and not int, provide decimal in bounds.  Also, I'm not comfortable with these bounds.  Granted 5 is really cold and 40 is really hot, I think we'd be safer with larger range.

Also, add description to state units at a minimum.